packages/test-app/src/components/SpaceForm.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useState } from 'react';
```

Comment:
These Space components aren't currently used, but will be the UI for configuring spaces in the future

Reply:
Nice!